assets/router/daemonset.yaml

```diff
@@ -0,0 +1,44 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: router
+  labels:
+    app: router
+spec:
+  selector:
+    matchLabels:
+      app: router
+  template:
+    metadata:
+      labels:
+        app: router
+    spec:
+      serviceAccountName: router
+      nodeSelector:
+        "node-role.kubernetes.io/infra": "true"
+      containers:
+        - name: router
+          image: "docker.io/openshift/origin-haproxy-router:v3.11.0"
```

Comment:
The router deliberately sets `imagePullPolicy: IfNotPresent` (https://github.com/openshift/origin/blob/b054a7bf5b11c760e5dd0f870fc852cc9b8b748b/pkg/oc/cli/admin/router/router.go#L905, https://github.com/openshift/origin/pull/1779#issuecomment-96407925), so I was wondering whether we should do the same.  It looks like `imagePullPolicy: IfNotPresent` is the default if the image tag is not "latest" (https://github.com/openshift/origin/blob/bce7b76872bbb233087ec0f9641c1f84e5115db0/vendor/k8s.io/kubernetes/pkg/apis/core/v1/defaults.go#L85-L86), so the default should be fine if we don't want to be explicit.

Reply:
Updated to set an explicit `imagePullPolicy` just because it's clearer